api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -362,6 +363,11 @@ public GenomicExtractionJob submitGenomicExtractionJob(DbWorkspace workspace, Db
       // Added in https://github.com/broadinstitute/gatk/pull/7698
       maybeInputs.put(EXTRACT_WORKFLOW_NAME + ".cohort_table_prefix", "\"" + extractionUuid + "\"");
     }
+    if (!Strings.isNullOrEmpty(cohortExtractionConfig.extractionFilterSetName)) {
```

Comment:
Since i have no clue about this code, wanted to check , if it would make sense to add a comment regarding what  would be the the result of adding key ".filter_set_name"